src/WPConfigTransformer.php

```diff
@@ -65,8 +65,8 @@ public function exists( $type, $name ) {
 		if ( ! trim( $wp_config_src ) ) {
 			throw new Exception( 'Config file is empty.' );
 		}
-
-		$this->wp_config_src = $wp_config_src;
+		// Normalize the newline to prevent an issue coming from OSX
```

Comment:
WPCS wants a period at the end of comments.
```suggestion
		// Normalize the newline to prevent an issue coming from OSX.
```